### ISSUE TITLE
IE Issue with "Check All" Checkbox

### ISSUE THREAD

**Issue by aetna-wsa** [Mottie/tablesorter#1365]
Hi, Rob,

I have a problem in that if you go (in IE11) to the problematic page, clicking the select all box does nothing. It does not check, nor does it check the checkboxes below it. I'm dynamically generating checkboxes based on server-side criteria. This means that there could be 9 rows with no checkbox, but row 10 does have one. Expected behavior would be that clicking on the 'select all' box it would select that child row's checkbox. 

Are you aware of any issues there?

**Comment by Mottie** [comment]
Hi @aetna-wsa!

I'll be back next week so I won't be able to check out this problem. In the mean time, would you please provide an example of the code or a demo showing this problem? Thanks.

**Comment by aetna-wsa** [comment]
I appreciate it. When next week do you plan to return?

Thanks,
Darren

Darren Shrager
UX Designer, WSA

“Design is directed toward human beings.
To design is to solve human problems by
Identifying them and executing the best
solution.”

  —Ivan Chermayeff

AetNet Spaces:
WSA<https://aetna.jiveon.com/community/wsa> | WSA-AD<https://aetna.jiveon.com/community/wsa/application-development> | Water Buffaloes<https://aetna.jiveon.com/groups/water-buffaloes/overview>

**Comment by Mottie** [comment]
Hi @aetna-wsa!

Oh, I just pushed v2.26.8, but I forgot about this issue... I tried [this demo](http://jsfiddle.net/Mottie/y5dmhybr/) and it appears that the header checkbox does behave as expected. 

Maybe I misread the question, but the code isn't really set up to deal with child rows that have checkboxes in a different column. Or, did you mean that the checkbox isn't updating the dynamically added checkbox? It should, but only if you "update" after adding the new rows.
